### PR TITLE
[Resprite] tgstation pill sprites

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

tgstation pills cool, use those :)

# Media

image

# Changelog

:cl: DEATHB4DEFEAT
- tweak: The chemists have suddenly decided to greatly improve the look of their pills, making patients much happier to ingest them.

### CODE REVIEW

**Review by Pspritechologist — COMMENTED** [review]
Does Eris have giant pills? 13 doesn't support smooth scaling as far as my understanding goes.

Are we normalizing drawing sprites at whatever resolution we desire, and simply scaling them? It's a bold choice...
Going to make for a very different vibe if that becomes the norm.

File: Resources/Prototypes/Entities/Objects/Specific/chemistry.yml [hunk 1]
```diff
@@ -272,6 +272,7 @@
   - type: Sprite
     sprite: Objects/Specific/Chemistry/pills.rsi
     state: pill
+    scale: 0.5, 0.5
```
Comment (Pspritechologist) [inline Resources/Prototypes/Entities/Objects/Specific/chemistry.yml]:
```suggestion
    scale: 0.35, 0.35
```
? Looked a little large in the preview.

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Entities/Objects/Specific/chemistry.yml]: I don't think the preview is scaled.

**Comment by DEATHB4DEFEAT** [comment]
I somehow scaled the sprites 2x

**Comment by Pspritechologist** [comment]
> I somehow scaled the sprites 2x

Somehow? 😨

**Review by CharcoalGungan — APPROVED** [review]
"Based and redpilled" (pill5.png) changes. Meta seems to have been made properly with the correct copyrights and meta shenanigans.

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**